app/containers/Products/ProductRowItemContainer.tsx

```diff
@@ -376,6 +389,9 @@ export const ProductRowItemComponent: React.FunctionComponent<Props> = ({
           opacity: 0.5;
           cursor: pointer;
         }
+        table td {
+          vertical-align: middle !important;
+        }
```

Comment:
```suggestion
        .table td {
          vertical-align: middle;
        }
```
You don't need the `!important`, just a higher specificity

Reply:
Alternatively, use the `.align-middle` class (which does use an `!important`, so I'm not sure how I feel about it)